web_monitoring/diffing_server.py

```diff
@@ -7,6 +7,7 @@
 import tornado.ioloop
 import tornado.web
 import web_monitoring
+import web_monitoring.differs
```

Comment:
Is the server currently working and deployed without this??? It seems to be required for me.

Reply:
I haven't re-deployed since the commit where we switched from string-based config to literal references to the modules.